scripts/packages/chocolatey/build.ps1

```diff
@@ -18,8 +18,11 @@ rm -force ./bazel.nuspec
 rm -force ./tools/chocolateyinstall.ps1
 
 Add-Type -A System.IO.Compression.FileSystem
-$zipDir = "$pwd/../../../output"
+$outputDir = "$pwd/../../../output"
 $zipFile = "$pwd/$($tvFilename)"
+$zipDir = "$outputDir/package"
+new-item $zipDir -type directory -ErrorAction SilentlyContinue
+cp "$outputDir/bazel.exe" $zipDir
```

Comment:
Let's not create garbage directories; use ZipArchive class to archive "$outputDir/bazel.exe".